python/pyarrow/tests/test_parquet.py

```diff
@@ -65,7 +65,7 @@ def datadir(datadir):
     "use_legacy_dataset", [True, pytest.param(False, marks=pytest.mark.skip)])
 
 
-def deterministic_row_order(use_legacy_dataset, chunk_size=None):
+def deterministic_row_order(use_legacy_dataset, chunk_size=-1):
```

Comment:
Only when actually passing it as None, it should be checked. So therefore giving the default another values as None.